pages/domains/domains_eligibility.md

```diff
@@ -50,9 +50,9 @@ When you request a .gov domain, we'll ask for information about your authorizing
 - [School districts](#school-districts)
 
 ### Executive branch federal agencies
-Domain requests from executive branch agencies must be authorized by CIOs or agency heads.
+Domain requests must be authorized by the agency's CIO or the head of the agency.
```

Comment:
My only comment is that I'd like the fed exec section to follow the pattern of the other sections by restating the org type. 

"Domain requests from executive branch federal agencies must..."

Reply:
good catch! will make that update.